sources/packages/web/src/components/layouts/student/StudentPageContainer.vue

```diff
@@ -2,6 +2,7 @@
   <full-page-container
     :layout-template="layoutTemplate"
     :full-width="fullWidth"
+    class="overflow-visible"
```

Comment:
Assuming this will be available for all student pages using the `student-page-container`, should we remove it from the `StudentAppealRequest.vue`?
I guess only pages that needd it would receive the `class` since it overrides the default vue component behavior.
Would be be better to have ot applied only to the page with the issue?

Reply:
yeah, I removed it from `StudentPageContainer` and I'm setting the class in `FullTimeApplication`. i just needed to add `v-bind="$attrs"` because `FullTimeApplication` is dealing with multiple root nodes: https://vuejs.org/guide/components/attrs#attribute-inheritance-on-multiple-root-nodes

Reply:
Reverted the v-bind as not needed. You were right. Thanks!